Mercado Libre as known instance of the Trusted Committer pattern

privero-meli opened the pull request:
Add Mercado Libre as known instance of the Trusted Committer pattern

privero-meli committed:
Add Mercado Libre as known instance of the Trusted Committer pattern

spier commented on patterns/2-structured/trusted-committer.md:
```diff
@@ -127,6 +127,10 @@ This has been tried and proven successful at:
+- Mercado Libre - We assign a section in the CONTRIBUTING file to inform who the TCs are.
+
+![image](https://user-images.githubusercontent.com/58035419/205659124-9fc7796d-3ee5-49b9-8c8c-1c7d24fa0488.png)
+
```
```suggestion
- Mercado Libre - adds a section in the `CONTRIBUTING.md` file to inform who the Trusted Committers are.

![image](https://user-images.githubusercontent.com/58035419/205659124-9fc7796d-3ee5-49b9-8c8c-1c7d24fa0488.png)
```

We try to stay away from using "we" in the Known Instances, as each of them is referring to a different company, which makes the use of "we" potentially confusing.

Also did some other rewording, please commit if you are happy with this.

spier reviewed (commented):
Left one comment inline.

Also would you mind adding the image to folder `/assets/img`.
Then we can add a relative link to that from the markdown file.

privero-meli committed:
Fix lint error

privero-meli committed:
Change description text

privero-meli committed:
Mercado Libre trusted committer image

privero-meli committed:
Add image to folder '/assets/img'

spier commented on patterns/2-structured/trusted-committer.md:
```diff
@@ -127,6 +127,9 @@ This has been tried and proven successful at:
 - PayPal
+- Mercado Libre - adds a section in the `CONTRIBUTING.md` file to inform who the Trusted Committers are.
+
+![Trusted Committer section](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section")
```
```suggestion
![Trusted Committer section in CONTRIBUTING.md of Mercado Libre](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section")
```

spier committed:
Extend label text

spier reviewed (commented):
@privero-meli just noticed one more thing:
If I understand it correctly, the image is meant to be a mock of the `CONTRIBUTING.md` section in use at mercadolibre. Right?

However the text uses the wording "working group", which may be confusing in this context?

Also noticed that the text is pretty close to what we use in [the our own TC file](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/252a506fa03ee8020d466f8e0196f4ec44a38cb4/TRUSTED-COMMITTERS.md#trusted-committers) in this repo.

So maybe something that went unnoticed when copying the text?

spier commented on patterns/2-structured/trusted-committer.md:
```diff
@@ -127,6 +127,9 @@ This has been tried and proven successful at:
 - PayPal
+- Mercado Libre - adds a section in the `CONTRIBUTING.md` file to inform who the Trusted Committers are.
+
+![Trusted Committer section in CONTRIBUTING.md of Mercado Libre](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section")
```
```suggestion
![Trusted Committer section in CONTRIBUTING.md of Mercado Libre](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section in CONTRIBUTING.md of Mercado Libre")
```

spier committed:
Extend hover text for image

Extend hover text for image.
Note that this will only used in GitHub, not in the published gitbook.

spier reviewed (approved):
Looks awesome!

We could consider to turn the image from Mercado Libre into text (a quote maybe?).
That way others can just copy and paste it, rather than having to re-type it all.
However we can leave that for a future PR, if we like that idea.

Thank you for this great contribution @privero-meli!

spier committed:
Add slack inviter to ignore list, to fix link check
